.travis.yml

```
@@ -20,6 +26,9 @@ before_cache:
 
 after_success:
   - snyk/monitor.sh
+  - ./gradlew jacocoTestReport
```

```suggestion
  - ./gradlew jacocoRootReport
```

Make sure report aggregator gets run by Travis